core/test/invariants/LenderHarness.sol

```diff
@@ -161,7 +166,7 @@ contract LenderHarness {
         // Make sure `msg.sender` has enough assets to deposit
         if (amountToTransfer > 0) {
             vm.prank(msg.sender);
-            vm.expectRevert(bytes(shares > 0 ? "Aloe: insufficient pre-pay" : "Aloe: zero impact"));
+            vm.expectRevert(bytes(shares > 0 ? "TRANSFER_FROM_FAILED" : "Aloe: zero impact"));
```

Comment:
Forgot to update this when we added the `approve` flow in https://github.com/aloelabs/aloe-ii/pull/87